content/about/notes/1.1/index.md

```diff
@@ -1,32 +1,149 @@
+testing the release candidates at various scales.
+
+These release notes describe what's different between Istio 1.0 and Istio 1.1.
 
 {{< relnote_links >}}
 
-## Policies and Telemetry
+## Upgrades
+
+- **Helm Changes**. 
+TBD: need full content & links
+
+- things to watch out for/configure before you upgrade - from all subsystems below. For example, outbound traffic is allowed by default for unknown ports, what else?
+TBD: need full content & links
+  - Egress gateway is disabled by default
+  - Mixer policy is off by default
+  - Outbound traffic policy is set to ALLOW_ANY. Traffic to unknown ports will be forwarded as is.
+
+- **TBD: TITLE**, Control plane and data plane requested resource changes
+TBD: need full content & links
+
+## Traffic management
+
+- **New `Sidecar` Resource**. The `Sidecar` resource allows you to limit the set of services that are visible to sidecar proxies in a given
+namespace, thereby reducing amount of configuration computed and transmitted to the proxy. On large clusters, we recommend adding
+a sidecar object per namespace. TBD: links
+
+- **Restrict Visibility of Networking Resources**. The new `exportTo` field has been added to all the networking resources.
+It currently takes only `.` (same namespace as resource) or `*` (default, all namespaces). Set the field to `.` to make the
+`DestinationRule` or `VirtualService` to be visible only within its own namespace. 
+
+- **Updates to `ServiceEntry` Resources**. Added the ability to specify the locality where the endpoint resides as well as
+the SAN associated with the service for use with mutual TLS. Also, service entries with HTTPS ports no longer need an additional
+virtual service to enable SNI-based routing.
+
+- **Locality-Aware Routing**. Istio now fully supports routing to endpoints in the same locality before picking endpoints in other localities.
+
+- **Refined Multicluster Routing**. The multicluster setup has been simplified and additional deployment modes have been enabled. It is now possible to connect multiple
+clusters simply using their ingress gateways without needing pod-level VPNs, deploy control planes in each cluster for HA, and span a namespace across several clusters
+to create global namespaces. Locality-aware routing is enabled by default in the HA control plane solution.
 
-- **Kiali**. The Service Graph addon has been [deprecated](https://github.com/istio/istio/issues/9066) in favor of [Kiali](https://www.kiali.io). See the [Kiali Task](/docs/tasks/telemetry/kiali/) for more details about Kiali.
+- **Istio Ingress Deprecated**. Previously deprecated Istio ingress has been removed. Refer to the
+[Securing Kubernetes Ingress with Cert-Manager](/docs/examples/advanced-gateways/ingress-certmgr/) example for more details on how
+to use Kubernetes Ingress resources with [gateways](/docs/concepts/traffic-management/#gateways).
 
 ## Security
```

Comment:
How about adding https://github.com/istio/istio/pull/11667 as well? As this is also a significant feature for security which can enable end user set different CA and Certs for different namespaces.